tools/mmp/driver.cs

```diff
@@ -1495,6 +1499,22 @@ static void CreateSymLink (string directory, string real, string link)
 
 				CopyResourceFile ("config", "config");
 			}
+
+			string machineConfigDestDir = Path.Combine (mmp_dir, "mono/4.5/");
+			string machineConfigDestFile = Path.Combine (machineConfigDestDir, "machine.config");
+			if (custom_machine_config_path != null) {
+				if (!File.Exists (custom_machine_config_path))
+					throw new MonoMacException (97, true, "machine.config file '{0}' can not be found.", custom_machine_config_path);
+				CreateDirectoryIfNeeded (machineConfigDestDir);
+				File.Copy (custom_machine_config_path, machineConfigDestFile);
+
+			}
+			else if (!disable_empty_machine_config) {
```

Comment:
For 95% of the use cases, things have worked fine without one (or the garbage path we did last release). I'm just concerned given the large nature of the BCL means that somehow the addition of one will break something. 

I might be paranoid though. I guess I can nuke it.